src/NCronJob/Execution/JobExecutor.LogMessage.cs

```diff
@@ -4,18 +4,18 @@ namespace NCronJob;
 
 internal sealed partial class JobExecutor
 {
-    [LoggerMessage(LogLevel.Debug, "Running job: '{JobType}' with correlation id: {CorrelationId}.")]
-    private partial void LogRunningJob(Type jobType, Guid correlationId);
+    [LoggerMessage(LogLevel.Debug, "Running job: '{JobName}' with correlation id: {CorrelationId}.")]
```

Comment:
See discussion - I do think adding the jobtype might help. Eventhough the job name is unique, you might still have to go into code to have the full picture.

Reply:
Fixed.